wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -1300,6 +1327,184 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:complexType>
 			</xs:element>
 			<!--===============================-->
+			<!--============AudioClipManagement===================-->
+			<xs:complexType name="AudioClipConfiguration">
+				<xs:sequence>					
+					<xs:element name="Enabled" type="xs:boolean">
+						<xs:annotation>
+							<xs:documentation> Enables/disables the audio clip.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="Name" type="xs:string">
+					<xs:annotation>
+						<xs:documentation> Name of the audio clip.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="Token" type="tt:ReferenceToken">
+					<xs:annotation>
+						<xs:documentation> Unique Token associated with the audio clip.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="AudioOutputToken" type="tt:ReferenceToken">
+					<xs:annotation>
+						<xs:documentation> Token of the physical Audio Output.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="Type" type="xs:string">
+					<xs:annotation>
+						<xs:documentation> Format of the audio clip. For definitions see tt:SupportedAudioClipFormat parameter.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="RepeatCycles" type="xs:int">
+					<xs:annotation>
+						<xs:documentation> Number of times the audio clip can be repeated when it is played. The value 0 means no repeat i.e. play the audio clip, does not repeat after the first play. 1 means play the audio clip and repeat once after the first play. -1 means repeat the audio clip infinite after the first play until the clip is stopped by the user operation.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="Gain" type="xs:int" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation> Optional Gain to control volume of the audio clip when it is played.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="ScheduleToken" type="tt:ReferenceToken" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation> Optional Schedule Token associated with the audio clip configuration. Audio clip can be played when configured schedule is triggered.</xs:documentation>
+					</xs:annotation>
+					</xs:element>
+					<xs:element name="PlayStatus" type="xs:boolean" minOccurs="0">
```

Comment:
PlayStatus is state, split to seperate structure and/or an status event?

Reply:
audio clip playback itself is a action of schedule or event trigger. Raising one more status event for audio clip playback is redundant.

Reply:
PR is updated by listing PlayStatus only in GetAudioClips API Response.